drivers/google/option.go

```diff
@@ -5,11 +5,10 @@
 package google
 
 import (
+	"github.com/drone/autoscaler/drivers/internal/userdata"
```

Comment:
hey there, I recommend reverting back.  The Go style guide recommends that imports should be grouped by standard library first, then application-specific imports, and then third party imports.

Reply:
Sorry for once I relied to use Goland but I messed things up rebasing. Fixing it now :)

Reply:
Here, sorry. I don't know how I got myself into that state but all sorted.